core/domain/stats_domain.py

```diff
@@ -882,7 +1007,9 @@ def __init__(
         self.schema_version = schema_version
         self.is_valid = is_valid
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> Any:
```

Comment:
Why does this return Any? Also, same question as above.

Reply:
Done